flask_app/__init__.py

```diff
@@ -6,6 +6,15 @@
 
 dictConfig(LOGGING_CONFIG)
 
+logging.getLogger().setLevel(0)  # Allows all messages to go through logger, msg to be filtered at handlers
```

Comment:
Are we setting the level manually here?

Reply:
Logger objects take in the logging message and pass it to the handler object, whereby the handler prints the message. Both the logger and handler objects have filters that remove messages below their level threshold. In this case, I want only the handler to filter the messages, so I set the logger's level to be 0 to allow all messages through the logger. Otherwise, all messages below DEBUG (10), will not be let through the logger (as the logger has a filter of 10)